src/sidebar/components/MarkdownView.tsx

```diff
@@ -42,9 +60,38 @@ export default function MarkdownView({
   }, [markdown]);
 
   useEffect(() => {
-    if (hasMentions) {
-      renderMentionTags(content.current!, mentions);
+    if (!hasMentions) {
+      return () => {};
     }
+
+    const listenerCollection = new ListenerCollection();
+    const foundMentions = renderMentionTags(content.current!, mentions);
+
+    listenerCollection.add(
+      content.current!,
+      'mouseenter',
+      ({ target }) => {
+        const element = target as HTMLElement;
+        const mention = foundMentions.get(element) ?? null;
+
+        if (mention) {
+          setPopoverContent(mention);
+          mentionsPopoverAnchorRef.current = element;
+        }
+      },
+      { capture: true },
+    );
+    listenerCollection.add(
+      content.current!,
+      'mouseleave',
+      () => {
+        setPopoverContent(null);
+        mentionsPopoverAnchorRef.current = null;
+      },
+      { capture: true },
+    );
```

Comment:
I need to check if we can use `onMouseEnterCapture` and `onMouseLeaveCapture` instead.

Reply:
I changed it